src/dosbox.cpp

```diff
@@ -796,6 +796,18 @@ void DOSBOX_Init()
 	                  "On 'auto' the mode is determined by 'sbtype'.\n"
 	                  "All OPL modes are AdLib-compatible, except for 'cms'.");
 
+	pstring = secprop->Add_string("opl_fadeout", when_idle, "off");
+	pstring->Set_help(
+	        "Fade out the OPL synth output after the last IO port write:\n"
+	        "  off:       Don't fade out; residual output will play forever (default).\n"
+	        "  on:        Wait 0.5s before fading out over a 0.5s period.\n"
+	        "  <custom>:  A custom fade out definition in the following format:\n"
+	        "               WAIT FADE\n"
+	        "             Where WAIT is how long before fading begins from 100 to 5000 in ms,\n"
+	        "             and FADE is the fade out period from 10 and 3000 in ms. Examples:\n"
+	        "                 300 200 (Wait 300ms before fading out over a 200ms period)\n"
+	        "                 1000 3000 (Wait 1s before fading out over a 3s period)");
```

Comment:
I'm not sure this is a good idea, just wondering if having to write `wait 300 fade 200` would be better or not. Whenever I have to specify a bunch of values I always forget what is what, so this would make it self-documenting at the setting level. But maybe it would unnecessary complication.

Reply:
We're not prefixing multi-values with their type.
For example:
- `sb_filter = lpf 2 12000` isn't: `lfp order 2 freq 12000`
- `sensitivity = 50 100` isn't `x 50 y 100`
- (and a couple others)

So I think using that approach here would be inconsistent with the other areas.

Reply:
Yeah, that's cool.